### DISCUSSION TITLE
I need to know the lowest version of Quarkus which supports JDK17??

### DISCUSSION THREAD

**Discussion by saytoshivam** [quarkusio/quarkus#33732] [Q&A]
I traversed everywhere but getting answer like "u can create  quarkus new project with jdk17".
I need to know the lowest version of quarkus which will support jdk17.
Thanks in advance.

**Comment by maxandersen** [comment]
What do you mean lowest number that support 17?

Supported as in which there still will be made security fixes for or when quarkus started to work with java 17 or when we officially started making java 17 the default for proejct creation ?
